packages/material-ui/src/ButtonGroup/ButtonGroup.js

```diff
@@ -12,6 +12,10 @@ import Button from '../Button';
 Button.styles;
 
 export const styles = theme => ({
+  /* Styles applied to the root element if `orientation="vertical"`. */
+  vertical: {
```

Comment:
By convention, we keep the root style rule first.

Reply:
Got it, thank you @oliviertassinari